lib/generators/rolemodel/modals/templates/app/views/application/_confirm.html.slim

```diff
@@ -0,0 +1,12 @@
+#confirm.modal-wrapper.flex.items-center.justify-center
```

Comment:
Can this be refactored to use the confirm dialog styles from Optics?

https://docs.optics.rolemodel.design/?path=/docs/popup-content-components-confirm-dialog--docs